### ISSUE TITLE
The size attribute with file input type doesn't seem to be working

### ISSUE THREAD

**Issue by vivek0460** [joecwallace/jquery-validator#34]
Hey.
Thanks for such an awesome validator. I have been using this for quite a while now & quite happy about this.

Currently, I am working on a project which needs client side file size validations. I went ahead & used the size validation rule.
It doesn't seem to be working the way it is supposed to.

Is this a known issue? What do you recommend to use?

Thanks.

**Comment by joecwallace** [comment]
You're right. It doesn't work. I will try to look at this soon, but I don't have a workaround right now.

If you come up with a fix and want to submit a pull request, that would be awesome.
